websites/O/OG Network/presence.ts

```diff
@@ -0,0 +1,77 @@
+const presence = new Presence({ clientId: "1237999667023708231" }),
+	assets = {
+		logo: "https://www.og-network.net/assets/logo.png",
+	};
```

Comment:
Logo's must be 512x512 as per [our guideliness](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md#logo).